test/interactions/panZoomInteractionTests.ts

```diff
@@ -1,39 +1,119 @@
+
+      beforeEach(() => {
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+
+        panZoomInteraction = new Plottable.Interactions.PanZoom();
+      });
+
+      it("can set scales", () => {
+        assert.deepEqual(panZoomInteraction.xScales(), [], "no xScales exist by default");
+        assert.deepEqual(panZoomInteraction.yScales(), [], "no yScales exist by default");
+
+        assert.strictEqual(panZoomInteraction.addXScale(xScale), panZoomInteraction, "setting the xScale returns the interaction");
+        assert.strictEqual(panZoomInteraction.addYScale(yScale), panZoomInteraction, "setting the yScale returns the interaction");
+
+        assert.deepEqual(panZoomInteraction.xScales(), [xScale], "xScale has been added");
+        assert.deepEqual(panZoomInteraction.yScales(), [yScale], "yScale has been added");
+      });
+
+      it("can remove scales", () => {
+        panZoomInteraction.addXScale(xScale);
+        panZoomInteraction.addYScale(yScale);
+
+        assert.deepEqual(panZoomInteraction.xScales(), [xScale], "xScale has been added");
+        assert.deepEqual(panZoomInteraction.yScales(), [yScale], "yScale has been added");
+
+        assert.strictEqual(panZoomInteraction.removeXScale(xScale), panZoomInteraction, "removing the xScale returns the interaction");
+        assert.strictEqual(panZoomInteraction.removeYScale(yScale), panZoomInteraction, "removing the yScale returns the interaction");
+
+        assert.deepEqual(panZoomInteraction.xScales(), [], "xScale has been removed");
+        assert.deepEqual(panZoomInteraction.yScales(), [], "yScale has been removed");
+
+      });
+
+      it("can add set the xScales in batch", () => {
+        let xScale2 = new Plottable.Scales.Linear();
+
+        panZoomInteraction.addXScale(xScale);
+        panZoomInteraction.addXScale(xScale2);
+        let xScales = panZoomInteraction.xScales();
+
+        panZoomInteraction.xScales([]);
+        assert.deepEqual(panZoomInteraction.xScales(), [], "scales can be removed with the xScales call");
+
+        panZoomInteraction.xScales([xScale, xScale2]);
+        assert.deepEqual(xScales, panZoomInteraction.xScales(), "setting and adding x scales result in the same behavior");
+      });
+
+      it("can add the yScales in batch", () => {
+        let yScale2 = new Plottable.Scales.Linear();
+        panZoomInteraction.addYScale(yScale);
+        panZoomInteraction.addYScale(yScale2);
+        let yScales = panZoomInteraction.yScales();
+
+        panZoomInteraction.yScales([]);
+        assert.deepEqual(panZoomInteraction.yScales(), [], "scales can be removed with the yScales call");
+
+        panZoomInteraction.yScales([yScale, yScale2]);
+        assert.deepEqual(yScales, panZoomInteraction.yScales(), "setting and adding y scales result in the same behavior");
+      });
+
+      it("silently fails on adding an already existent xScale", () => {
```

Comment:
this title is misleading... I don't think it "fails silently", it just doesn't add duplicate